object-manipulation-1/src/get-value.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable no-unused-vars */
+
+function getValue(object, key) {
+  var value;
+  for (var property in object) {
+    value = object[key];
+  }
```

Comment:
You can do this without a loop. Just access the property `key` on the `object` and return it.